sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/MediaStreamingAudioInternal.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation.implementation.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The MediaStreamingAudioInternal model. */
+public final class MediaStreamingAudioInternal {
+
+    /*
+     * The audio data.
+     */
+    @JsonProperty(value = "data")
+    private byte[] audioData;
+
+    /*
+     * The timestamp of when the media was sourced.
+     */
+    @JsonProperty(value = "timestamp")
+    private String timestamp;
+
+    /*
+     * The participantId.
+     */
+    @JsonProperty(value = "participantRawID")
+    private String participantRawID;
```

Comment:
This also should return a `CustomerIdentifier` object, and not the actual raw id.
@Ninika could you please confirm?

Reply:
@cochi2 This is the internal class only used for deserializing the package and is not exposed to customers. The MediaStreamingAudio class is the one Im exposing and in that one I'm reading the raw ID into a CommunicationIdentifier